### PR TITLE
[EdgeTPU] Update the testing codes to register EdgeTPUToolchain 

### PR DESCRIPTION
**Pull request by rhqjatn2398**
This draft modify the testing codes to test registration of EdgeTPUToolchain.
Also, This commit exports BackendMap interface to use it in the testing codes.

ONE-vscode-DCO-1.0-Signed-off-by: Bumsoo Ko <rhqjatn2398@naver.com>

### CODE REVIEW

File: src/Tests/Toolchain/ToolchainProvider.test.ts [hunk 1]
```diff
@@ -34,28 +34,22 @@ import {
...
   });
 
   teardown(function () {
-    Object.keys(gToolchainEnvMap).forEach(
-      (key) => delete gToolchainEnvMap[key]
-    );
+    if (gToolchainEnvMap[backendName] !== undefined) {
+      delete gToolchainEnvMap[backendName];
+    }
   });
```
Comment (profornnan) [inline src/Tests/Toolchain/ToolchainProvider.test.ts]: I think teardown function should be located at the bottom of "Toolchain" test suite :-D
(related to https://github.com/Samsung/ONE-vscode/pull/1686#discussion_r1336789547)

File: src/Tests/View/InstallQuickInput.test.ts [hunk 2]
```diff
@@ -46,30 +45,22 @@ suite("View", function () {
...
     });
 
     teardown(function () {
-      Object.keys(gToolchainEnvMap).forEach(
-        (key) => delete gToolchainEnvMap[key]
-      );
+      if (gToolchainEnvMap[backendName] !== undefined) {
+        delete gToolchainEnvMap[backendName];
+      }
     });
```
Comment (profornnan) [inline src/Tests/View/InstallQuickInput.test.ts]: Same here

**Review by profornnan — CHANGES_REQUESTED** [review]

File: src/Tests/Toolchain/ToolchainProvider.test.ts (hunk 1 above)
Comment (rhqjatn2398) [reply src/Tests/Toolchain/ToolchainProvider.test.ts]: Oh, Yes.
Thanks for reminding!

I'll fix the codes. 😃

File: src/Tests/View/InstallQuickInput.test.ts (hunk 2 above)
Comment (rhqjatn2398) [reply src/Tests/View/InstallQuickInput.test.ts]: 👍

**Review by profornnan — APPROVED** [review]
LGTM 👍

**Comment by rhqjatn2398** [comment]
These commits are cherry-picked on #81

**Pull request closed without merging**